post-processor/googlecompute-export/post-processor.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/hashicorp/packer/builder/googlecompute"
+	googlecompute "github.com/hashicorp/packer/builder/googlecompute"
```

Comment:
Was this automatically changed by your editor?

Reply:
No, I did this. Thats to clearly identify internal code import like inside the aws post processor

Reply:
Well, I seems like its unnecessary. I'll remove it